docs/pull_request_template.md

```diff
@@ -4,15 +4,15 @@
 
 # Checklist
 
+Be sure you've read README.md and understand the scope of this repo.
+
 If you're unsure about a box, leave it unchecked. A maintainer will help you.
 
-- [ ] I understand README.md. I also understand that acceptance of
-  community PRs will be delayed until the test and CI systems are online.
 - [ ] Identifiers in product code changes are properly `_Ugly` as per
   https://eel.is/c++draft/lex.name#3.1 or there are no product code changes.
 - [ ] The STL builds successfully and all tests have passed (must be manually

```

Comment:
Does it even make sense to have this "all tests have passed" bit here until we have tests? Why not simply "The STL builds successfully" which is at least actionable for externals?

Reply:
(Consider this a suggested change rather than opening a debate: it's silly for us to invest time discussing at length things we should be investing our time into actually fixing.)

Reply:
>Why not simply "The STL builds successfully" which is actionable for externals?

The purpose of this checkbox is for one of us to check after we have run Contest.

Reply:
Billy and I discussed this offline, and I've convinced him this is useless and unrelated to this PR.